dev_apps/music-nga/elements/music-artwork.js

```diff
@@ -178,6 +213,14 @@ proto.attributeChangedCallback = function(attr, oldVal, newVal) {
   }
 };
 
+proto.nextRepeat = function() {
```

Comment:
Yeah, I saw that. I didn't want to cross-pollenate between a web component and the backend though.